tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_stock.feature

```diff
@@ -0,0 +1,88 @@
+      | product1MBlue  | Size - M, Color - Blue  | [Size:M,Color:Blue]  | 0               | 0           | 0        | false      |
+    And combination "product1SBlack" should have following stock details:
+      | combination stock detail   | value |
+      | quantity                   | 0     |
+      | minimal quantity           | 1     |
+      | low stock threshold        | 0     |
+      | low stock alert is enabled | false |
+      | location                   |       |
+      | available date             |       |
+    When I update combination "product1SBlack" stock with following details:
+      | quantity                   | 100         |
+      | minimal quantity           | 10          |
+      | location                   | Storage nr1 |
+      | low stock threshold        | 10          |
+      | low stock alert is enabled | true        |
+      | available date             | 2021-10-10  |
+    Then combination "product1SBlack" should have following stock details:
+      | combination stock detail   | value       |
+      | quantity                   | 100         |
+      | minimal quantity           | 10          |
+      | low stock threshold        | 10          |
+      | low stock alert is enabled | true        |
+      | location                   | Storage nr1 |
+      | available date             | 2021-10-10  |
+    When I update combination "product1SBlack" stock with following details:
+      | quantity            | 100         |
+      | minimal quantity    | 1           |
+      | location            | Storage nr2 |
+      | low stock threshold | 10          |
+    Then combination "product1SBlack" should have following stock details:
+      | combination stock detail   | value       |
+      | quantity                   | 100         |
+      | minimal quantity           | 1           |
+      | low stock threshold        | 10          |
+      | low stock alert is enabled | true        |
+      | location                   | Storage nr2 |
+      | available date             | 2021-10-10  |
+    When I update combination "product1SBlack" stock with following details:
+      | quantity                   | 0          |
+      | minimal quantity           | 0          |
+      | location                   |            |
+      | low stock threshold        | 0          |
+      | low stock alert is enabled | false      |
+      | available date             | 2020-01-01 |
+    Then combination "product1SBlack" should have following stock details:
+      | combination stock detail   | value      |
+      | quantity                   | 0          |
+      | minimal quantity           | 0          |
+      | low stock threshold        | 0          |
+      | low stock alert is enabled | false      |
+      | location                   |            |
+      | available date             | 2020-01-01 |
```

Comment:
What about failures?
```
    When I update combination "product1SBlack" stock with following details:
      | quantity                   | false          |
      | minimal quantity           | nothing          |
      | location                   |            |
      | low stock threshold        | amzodkamzokdmokazd          |
      | low stock alert is enabled | null      |
      | available date             | 2021-01-01 is not a date |
```

I think testing wrong behaviour should be done (in a new PR of course)